app/src/main/java/org/simple/clinic/searchresultsview/PatientSearchView.kt

```diff
@@ -68,6 +68,7 @@ class PatientSearchView(context: Context, attrs: AttributeSet) : RelativeLayout(
 
     val screenDestroys = detaches().map { ScreenDestroyed() }
     setupScreen(screenDestroys)
+    isSaveEnabled = false
```

Comment:
Do we still need to override `onSaveInstanceState` and `onRestoreInstanceState` if we're disabling state restoration?

Reply:
This is only temporary and prevents them from being invoked. Once we add Paging v3, we will remove this and let this be saved and restored as usual.